.github/workflows/kind.yaml

```diff
@@ -22,7 +22,13 @@ jobs:
   installation-and-connectivity:
     runs-on: ubuntu-22.04
     timeout-minutes: 40
+    strategy:
+      matrix:
+        mode: ["classic", "helm"]
```

Comment:
Am I correct in that later on we will drop the classic mode and only have Helm available?

Reply:
yeah we'll gradually transition to the helm mode, and then deprecate the classic mode ✨